docker-unified/Dockerfile

```diff
@@ -0,0 +1,269 @@
+###############################################
+#                 Build stage                 #
+###############################################
+FROM alpine:3.16 AS web-setup
```

Comment:
Why not just latest?

Reply:
Usually we pin versions to minimize pipelines breaking without us making a change, but we can probably use `latest` here.